music_install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+ # Install current MUSIC version.
+git clone https://github.com/INCF/MUSIC.git music
+cd music
+./autogen.sh
+./configure --prefix=$HOME/.cache/music.install
+make
+make install
```

Comment:
Instead of excluding the `music` directory from copyright header checks below, I think it's cleaner to `rm -rf` the source directory after the `make install` step here.